examples/fewshot_protonet/configs/demo.yaml

```diff
@@ -0,0 +1,9 @@
+# This is a demo config file for protonet.
+# Users can modify the dataset and output paths to train their own model.
+# For more adjustable hyperparameters, please refer to template.yaml.
+DATASET:
+  ROOT: 'data/omniglot'
```

Comment:
Please check and fix all. We use double quotes for strings. See other examples of PyKale.